### DISCUSSION TITLE
How to properly set up routing with shadowsocks to let ss client access LAN clients?

### DISCUSSION THREAD

**Discussion by curiousbarbarian** [shadowsocks/shadowsocks-org#271] [General]
Hello, folks.

I have a VPS serving shadowsocks and wireguard server. I have an iPhone with shadowrocket app, that allows to connect to wireguard/shadowsocks server. I also have a MikroTik router that is connected to VPS using wireguard and serves as gateway to behind-the-router LAN (192.168.0.0/24). On VPS I have a route to access this LAN using WG peer MikroTik as gateway (in wg0.conf MikroTik AllowedIPs contains 192.168.0.0/24).

So I need to have access to LAN behind the router from my iPhone 24/7. In case of wireguard it works without a problem. I connect my phone to VPS and have access to 192.168.0.0/24. But if I connect my phone using shadowsocks I don't have this access.

What should I do with shadowsocks or VPS or whatever to have access to 192.168.0.0/24 from my phone that is connected to VPS with shadowsocks?

**Comment by buttatvkrabi-ui** [comment]
https://github.com/shadowsocks/shadowsocks-org/discussions/271#discussion-7048831

**Comment by buttatvkrabi-ui** [comment]
https://github.com/shadowsocks/shadowsocks-org/discussions/271#discussion-7048831
